Sources/Logging/Logging.swift

```diff
@@ -1171,11 +1171,16 @@ public struct StreamLogHandler: LogHandler {
 
 /// No operation LogHandler, used when no logging is required
 public struct SwiftLogNoOpLogHandler: LogHandler {
+    
     public init() {}
 
     public init(_: String) {}
 
     @inlinable public func log(level: Logger.Level, message: Logger.Message, metadata: Logger.Metadata?, file: String, function: String, line: UInt) {}
+    
+    @inlinable public func log(level: Logger.Level, message: Logger.Message, metadata: Logger.Metadata?, source: String, file: String, function: String, line: UInt) {
```

Comment:
in other words, this specific forcing of the requirement impl breaks compilation of any LogHandler implementation that was implementing the 1.0 API (the one above), so we can't force such change without making a major version bump of swift-log.